libs/ui/src/components/NeoStickyModal/NeoStickyModal.scss

```
@@ -0,0 +1,44 @@
+@import '../../scss/variable.scss';
+// mobile mixin
+@import "bulma/sass/utilities/_all";
+
+.confirm-modal {
+  @include mobile {
+    position: fixed !important;
+    bottom: 0 !important;
+    left: 0 !important;
+    right: 0 !important;
+    border-radius: 0.75rem 0.75rem 0 0 !important;
+
+    &.mobile-modal-height {
+      height: 90vh;
+    }
+  }
+
+  &.only-top-border {
+    border-right: 0 !important;
+    border-left: 0 !important;
+    border-bottom: 0 !important;
```

border-<direction>: none !important;

---

need border top

---

sorry i was unclear
i meant
```suggestion
    border-right: none !important;
    border-left: none !important;
    border-bottom: none !important;
 ```
 
 it's a small code style issue, but none is more sensible then 0 in this context. don't you agree?